integrations/api_repo_test.go

```diff
@@ -10,6 +10,7 @@ import (
 
 	"code.gitea.io/gitea/models"
 
+	api "code.gitea.io/sdk/gitea"
```

Comment:
Hm this was gofmt's choice because I aliased it. I'll edit it for the styleguide though